biscuits.pyx

```diff
@@ -32,9 +32,20 @@ cpdef str unquote(str input):
             except ValueError:
                 return None
             i += 2
-        elif current == '\\' and i+1 < length and input[i+1] in ('"', '\\'):
-            output.append(input[i+1])
-            i += 1
+        elif current == '\\':
+            if i+1 < length:
+                next = input[i+1]
+                if next in ('"', '\\'):
+                    current = next
+                    i += 1
+                elif next in ('0', '1', '2', '3'):  # Octal escaped char.
+                    next = input[i+1:i+4]
+                    i += 3
+                    try:
+                        current = chr(int(next, 8))
+                    except ValueError:
+                        return None
```

Comment:
Not sure at the end if we want to skip the value like this (this is what `return None` does), or maybe just return the raw bytes (without escaping them).
Same above.
I'd keep the behaviour for now, but let's think about it.